Resources/Audio/Weapons/attributions.yml

```diff
@@ -19,3 +19,7 @@
   copyright: "User Mystovski on freesound.org. Modified by LankLTE on github."
   source: "https://freesound.org/people/Mystovski/sounds/201111/"
 
+- files: ["star_hit.ogg"]
+  license: "CC0-1.0"
+  copyright: "Based on genhit3.ogg, modified by deltanedas on github."
```

Comment:
is this correct thing to do

Reply:
sure as long as the original is CC0-1.0

Reply:
yeah but it doesnt say the license anywhere so what do i do

Reply:
@metalgearsloth please clarify

Reply:
Just use the license for genhit3.ogg which is probably on TG.